app/css/obsadmin.css

```diff
@@ -489,16 +489,18 @@ li#location.selected a{
   float: none;
   box-shadow: none;
 }
+.full-size{
+  width: 100%;
```

Comment:
Keep one space between classes

Reply:
Thanks for the heads up I somehow missed that, fixing it right away.